src/base/utils/string.cpp

```diff
@@ -33,10 +33,13 @@
 
 #include <QCollator>
 #include <QLocale>
-#include <QRegExp>
```

Comment:
This should remain as noted in the comment below. Also it should move to the QT version check conditionals.

Reply:
>This should remain as noted in the comment below. 

What comment do you mean?
Doesn't seem like this header is really used.

Reply:
Ah, I see `QRegExp::Wildcard` here.

Reply:
All requested changes are done.